cmd/dbseed/main.go

```diff
@@ -154,8 +156,8 @@ func seedData(config *viper.Viper) {
 
 	// Seed a plan with some documents
 	planWithDocuments := createModelPlan(store, logger, "Plan with Documents", "MINT")
-	_ = planDocumentCreate(store, logger, s3Client, planWithDocuments, "File (Unscanned)", "cmd/dbseed/data/sample.pdf", "application/pdf", models.DocumentTypeConceptPaper, false, nil, nil, false, false)
-	_ = planDocumentCreate(store, logger, s3Client, planWithDocuments, "File (Scanned - No Virus)", "cmd/dbseed/data/sample.pdf", "application/pdf", models.DocumentTypeMarketResearch, false, nil, zero.StringFrom("Company Presentation").Ptr(), true, false)
+	restrictedDocument := planDocumentCreate(store, logger, s3Client, planWithDocuments, "File (Unscanned)", "cmd/dbseed/data/sample.pdf", "application/pdf", models.DocumentTypeConceptPaper, true, nil, nil, false, false)
```

Comment:
I think this is a little problematic, the seeding works, but it's creating data we wouldn't expect or want in reality. 

EG, you are linking a document that belongs to another model to a solution for a different model. 

I think it's ok that the resolver doesn't verify that  the document already belongs to the same model as the solution for now, as that should be controlled by the Front End (though it could be something we add in the future to harden the backend).

Could you update the seed data please so the need and solution are added onto `planWithDocuments` instead of `sampleModelPlan` please? Alternatively, you could make another model plan to demonstrate the document solution link.

Reply:
done 👍